client/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH0_DOMAIN=stampdcert.auth0.com
```

Comment:
This env file should be removed. The other PR covers this issue so This PR will not be merged.